pipenv/utils.py

```diff
@@ -950,11 +950,12 @@ def split_section(input_file, section_suffix, test_function):
     lockfile_sections = ('default', 'develop')
     if any(section in input_file for section in pipfile_sections):
         sections = pipfile_sections
-    elif any(section in input_file for section in lockfile_sections):
-        sections = lockfile_sections
-    # return the original file if we can't find any pipfile or lockfile sections
     else:
-        return input_file
+        if any(section in input_file for section in lockfile_sections):
```

Comment:
Why did this need to be moved?

Reply:
Hm I think I misunderstood what you were asking for in your review where you asked me to move this into the else statement for readability. To clarify -- did you mean that you prefer to test something like `if not any(section in input_file for section in pipfile_sections+lockfile_sections):` and do the alternative checks afterward?

Reply:
Sorry, I was simply asking that you move the comment into the else statement rather than having it out of alignment. There should need be any code changes.

Reply:
kk sorry, I thought the request was odd but it seemed simple enough so i just did (what i thought was) it